modules/configuration/pages/templating.adoc

```diff
@@ -300,11 +302,17 @@ A name to identify the test.
 
 *Type*: `string`
 
+=== `tests[].label`
 
-=== `tests[].config`
+A label to assign to this template when running the test.
+
+*Type*: `string`
 
-A configuration to run this test with, the config resulting from applying the template with this config will be linted.
+*Default*: `""`
+
+=== `tests[].config`
 
+A configuration to run this test against. Redpanda Connect checks the results of applying the template with this configuration using a linter.
```

Comment:
> Redpanda Connect checks the results of applying the template with this configuration using a linter.

`rpk connect template lint`: https://docs.redpanda.com/redpanda-connect/configuration/templating/#tests

Maybe we should link to that?